PVTPackage/source/MultiphaseSystem/PhaseSplitModel/DeadOilFlash.cpp

```diff
@@ -71,12 +73,12 @@ DeadOil_PhaseModel const & DeadOilFlash::getOilPhaseModel() const
 
 DeadOil_PhaseModel const & DeadOilFlash::getGasPhaseModel() const
 {
-  return m_gasPhaseModel;
+  return *m_gasPhaseModel;
 }
 
 BlackOil_WaterModel const & DeadOilFlash::getWaterPhaseModel() const
 {
-  return m_waterPhaseModel;
+  return *m_waterPhaseModel;
 }
```

Comment:
Can you add a small warning telling that one has to be careful when calling these member functions, because the result can be "void", so one has to be consistent?

Reply:
Done